geth/api/api.go

```diff
@@ -149,50 +149,50 @@ func (api *StatusAPI) SendTransaction(ctx context.Context, args common.SendTxArg
 
 // CompleteTransaction instructs backend to complete sending of a given transaction
 func (api *StatusAPI) CompleteTransaction(id common.QueuedTxID, password string) (gethcommon.Hash, error) {
-	return api.b.txQueueManager.CompleteTransaction(id, password)
+	return api.b.TxQueueManager().CompleteTransaction(id, password)
 }
 
 // CompleteTransactions instructs backend to complete sending of multiple transactions
 func (api *StatusAPI) CompleteTransactions(ids []common.QueuedTxID, password string) map[common.QueuedTxID]common.TransactionResult {
-	return api.b.txQueueManager.CompleteTransactions(ids, password)
+	return api.b.TxQueueManager().CompleteTransactions(ids, password)
 }
 
 // DiscardTransaction discards a given transaction from transaction queue
 func (api *StatusAPI) DiscardTransaction(id common.QueuedTxID) error {
-	return api.b.txQueueManager.DiscardTransaction(id)
+	return api.b.TxQueueManager().DiscardTransaction(id)
 }
 
 // DiscardTransactions discards given multiple transactions from transaction queue
 func (api *StatusAPI) DiscardTransactions(ids []common.QueuedTxID) map[common.QueuedTxID]common.RawDiscardTransactionResult {
-	return api.b.txQueueManager.DiscardTransactions(ids)
+	return api.b.TxQueueManager().DiscardTransactions(ids)
```

Comment:
Not yours and optional, but since you are refactoring anyway. Is there any sense to call it TxQueue**Manager**? It's not that it manages multiple queues, does it?